sourcegraph/about
# Host data driven relationships with your codebases pdf

Closes #5609 and hosts the pdf.

### Test

1. Visit `/guides/data-driven-relationships-with-codebases.pdf`
2. Ensure the title of the pdf matches the meta in the ticket
